contents/handbook/company/1-1s.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 1-1s
+sidebar: Handbook
+showTitle: true
+---
+
+A lot of managers and team members find 1-1s very useful. This short FAQ is intended to help you get the most out of them. 
+
+## What are 1-1s and why should I do them?
+
+The most important part of any manager's role at PostHog is [setting context](/handbook/company/management). 1-1s can be valuable opportunity to do this by diving deeper into topics which may not be easy to cover in other contexts like GitHub or sprint planning. 
+
+Most people do a weekly or bi-weekly 1-1 with their manager, with a 30min minimum. Any less than this is too short or infrequent. 
+
+## Sounds easy enough - what’s the problem?
+
+It’s very easy for 1-1s to:
+- Not happen because you’re both too busy
+- Just be a regular status update on tasks or projects
+- Be a general chat without much direction
+
+None of these options are terribly useful for either party. 
+
+## Ok, so how do I have a great 1-1?
+
+If you are a manager, ask better questions! The 1-1 should be a conversation, not just a status update. Of course you will naturally touch on discuss what’s going on that week, but your 1-1 should ideally be person-centric rather than project-centric.
+
+Julie Zhuo suggests the following buckets of questions you might find useful in her excellent book ‘[The Making of a Manager](https://www.juliezhuo.com/book/manager.html)’:
+
+- **Identify**
+  - What’s top of mind for you right now?
+  - What priorities are you thinking about this week?
+  - What’s the best use of our time today?
+- **Understand**
+  - What does your ideal outcome look like?
```

Comment:
outcome of this meeting/project/career?

Reply:
Wasn't clear - should be about what you're currently working on, I've updated.